packages/customer-account-ui-extensions/src/components/PageBlock/PageBlock.ts

```diff
@@ -0,0 +1,34 @@
+import {createRemoteComponent} from '@remote-ui/core';
+import type {
+  MaybeConditionalStyle,
+  MaybeShorthandProperty,
+  Spacing,
+} from '../shared-components';
+
+type Background = 'transparent' | 'color1' | 'color2' | 'color3';
+
+export interface PageBlockProps {
+  /**
+   * Adjust the background.
+   */
+  background?: Background;
+  /**
+   * Adjust the padding.
+   *
+   * To shorten the code, it is possible to specify all the padding properties in one property.
+   *
+   *
+   * Examples:
+   *
+   * - `base` means blockStart, inlineEnd, blockEnd and inlineStart paddings are `base`
+   *
+   * - [`base`, `none`] means blockStart and blockEnd paddings are `base`, inlineStart and inlineEnd paddings are `none`
+   *
+   * - [`base`, `none`, `loose`, `tight`] means blockStart padding is `base`, inlineEnd padding is `none`, blockEnd padding is `loose` and  blockStart padding is `tight`
+   */
+  padding?: MaybeConditionalStyle<MaybeShorthandProperty<Spacing>>;
```

Comment:
Originally, this type was:

```
padding?: CSSShorthand<Spacing> | Responsive<CSSShorthand<Spacing>>;
```

However, `CSSShorthand` and `Responsive` has since [been removed](https://github.com/Shopify/ui-extensions/pull/365/files#diff-caeca760aafbe6b1565e10f0fe4dd403ac60588a909237fafe0568ca722cedf6) from checkout-ui-extensions. I updated this type using the newly available types from checkout-ui-extensions:

- `MaybeConditionalStyle`: padding may be conditional, as it [supports conditionals](https://github.com/Shopify/customer-account-web/blob/dc9cd669a3bba9e8a41d8bf6e9ee27b44e19d882/app/components/PageBlock/PageBlock.tsx#L15)
- `MaybeShorthandProperty`: padding can use shorthand that mirrors CSS functionality (see code comment)

Reply:
Same here. can we put this in a shared file? Like here: https://github.com/Shopify/checkout-web/blob/master/packages/checkout-ui-extensions/src/components/shared.ts#L251